rest-api-spec/src/main/resources/rest-api-spec/test/indices.sort/10_basic.yaml

```diff
@@ -0,0 +1,72 @@
+---
+"Index Sort":
+
+  - skip:
+      version: " - 5.99.99"
+      reason:  this uses a new feature that has been added in 6.0.0
+
+  - do:
+      indices.create:
+        index: test
+        wait_for_active_shards: 1
```

Comment:
We usually don't have this setting in these tests. If it isn't needed I'd drop it.